static/js/components/HomePage.jsx

```diff
@@ -87,17 +89,31 @@ const HomePage = () => {
       cols={{ xlg: 16, lg: 12, md: 10, sm: 6, xs: 4 }}
       margin={[15, 15]}
       onLayoutChange={LayoutChangeHandler}
+      draggableHandle=".dragHandle"
     >
       <div key="sourceList" className={styles.homePageWidgetDiv}>
+        <DragHandleIcon className={`${styles.dragHandle} dragHandle`} />
```

Comment:
Can these `DragHandleIcon`s be incorporated directly into the corresponding components?